### PR TITLE
Feature/groups

### PR DESCRIPTION
**Pull request by rathorevaibhav**
## Summary

Groups functionality added.

## Test Plan

Added

### CODE REVIEW

File: src/containers/Group/Group.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import { GET_GROUP, GET_GROUPS } from '../../graphql/queries/Group';
+import { UPDATE_GROUP, CREATE_GROUP, DELETE_GROUP } from '../../graphql/mutations/Group';
+
+import { FormLayout } from '../Form/FormLayout';
+import { ReactComponent as TagIcon } from '../../assets/images/icons/Tags/Selected.svg';
+import styles from './Group.module.css';
+import { Dropdown } from '../../components/UI/Form/Dropdown/Dropdown';
+import { useQuery } from '@apollo/client';
+
+export interface TagProps {
```
Comment (kurund) [inline src/containers/Group/Group.tsx:13]: GroupProps :)

File: src/containers/Group/Group.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import * as Yup from 'yup';
+import { Input } from '../../components/UI/Form/Input/Input';
+import { GET_GROUP, GET_GROUPS } from '../../graphql/queries/Group';
+import { UPDATE_GROUP, CREATE_GROUP, DELETE_GROUP } from '../../graphql/mutations/Group';
+
+import { FormLayout } from '../Form/FormLayout';
+import { ReactComponent as TagIcon } from '../../assets/images/icons/Tags/Selected.svg';
```
Comment (kurund) [inline src/containers/Group/Group.tsx:8]: TagIcon?

File: src/containers/Group/GroupList/GroupListCard/GroupListCard.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
```
Comment (kurund) [inline src/containers/Group/GroupList/GroupListCard/GroupListCard.tsx:1]: Should we abstract this to 'ListCard' component? We can use this for other listings too.

**Comment by rathorevaibhav** [comment]
@kurund done with feedbacks here. ready for review again

**Review by kurund — APPROVED** [review]
Looks good.

**Pull request merged**